_pages/security/dynamic-scanning.md

```diff
@@ -5,63 +5,38 @@ parent: Security
 
 [OWASP ZAP](https://www.owasp.org/index.php/OWASP_Zed_Attack_Proxy_Project) is the tool we use for doing automated vulnerability scanning.
 
-![zap home screen](../../assets/images/zap_home.png)
+![ZAP home screen](../../assets/images/zap_home.png)
 
-## Usage
+### Preface
 
-You will need a running application to test, which you will want to be as production-like as possible. The 'Attack' scripts shouldn't do anything destructive, but will cause a spike in requests and errors. **Inform your team and #infrastructure before running it on live (not locally-hosted) sites.**
+You will need a running application to test, which you will want to be as production-like as possible—ideally a staging environment. Running a scan can cause a spike in requests and errors, so **inform your team and [#infrastructure](https://18f.slack.com/messages/infrastructure/) if you are going to run it on a production site.**
 
 We gave an introduction to ZAP talk as part of our engineering tech talks series.
 
 <iframe width="420" height="315" src="https://www.youtube.com/embed/2Dp7pAvKHaM" frameborder="0" allowfullscreen></iframe>
 
 Slides and additional information available [here](https://github.com/18F/tech-talks/tree/master/vuln-scanning).
 
-To get started:
+Using the the [Quick Start](https://github.com/zaproxy/zap-core-help/wiki/HelpAddonsQuickstartQuickstart) is a good way to get a basic idea of what ZAP does.
 
-1. [Install the ZAP desktop app](https://github.com/zaproxy/zaproxy/wiki/Downloads).
-1. In the Quick Start tab, enter the URL of your application.
-    * While you will need to run against production for the official ATO, you can also try out ZAP running against a local server.
+### Automated scanning
 
-1. Click "Attack".
-    * You should see a bunch of URLs start to pop up in the Spider tab, then the Active Scan tab.
-    * This will take a while.
-    * There is a progress bar in the Spider and Active Scan tabs. You can click the <img class="inline" src="../../assets/images/zap_graph.png" alt="small graph icon"/> button in the latter to get more details.
-1. Wait for the "Progress" area to say say "Attack complete".
-1. Open the Alerts tab.
+[Compliance Viewer](https://compliance-viewer.18f.gov) runs ZAP scans regularly for sites that opt in. This [continuous monitoring](../../ato/walkthrough/#step-6--continuously-monitor-the-controls) is a step towards being able to do [continuous compliance](../../ato/types/#continuous-authorization).
```

Comment:
Let's also say something more about whether automated scanning is required - something like "To get an ATO, you're required to do some kind of dynamic vulnerability scanning. It's best if you can set up automated scanning for continuous compliance, but if you can't set this up (such as if you need authentication support right now), manual scans can be ok."